src/chrome/content/rules/LastPass.xml

```diff
@@ -1,11 +1,8 @@
-<ruleset name="LastPass.com" default_off="Breaks addon ( https://www.eff.org/r.3Dq )">
+<ruleset name="LastPass.com">
   <target host="lastpass.com" />
   <target host="*.lastpass.com" />
 
   <securecookie host="^(?:.+\.)?lastpass\.com$" name=".*"/>
 
-  <rule from="^http://(?:www\.)?lastpass\.com/" to="https://lastpass.com/"/>
-
-	<rule from="^http://forums\.lastpass\.com/"
-		to="https://forums.lastpass.com/" />
+  <rule from="^http:" to="https:" />
```

Comment:
I've done that deliberately. All the other sub-domains worked too apparently. The certificate name of forum.lastpass.com for instance says it applies to *.lastpass.com.